src/exchange/kraken-processor/subscribers/tradingpairs-subscriber.go

```diff
@@ -0,0 +1,70 @@
+package subscribers
+
+import (
+	"context"
+
+	"github.com/aopoltorzhicky/go_kraken/rest"
+	"github.com/omiga-group/omiga/src/exchange/kraken-processor/configuration"
+	"github.com/omiga-group/omiga/src/exchange/kraken-processor/mappers"
+	exchangeConfiguration "github.com/omiga-group/omiga/src/exchange/shared/configuration"
+	"github.com/omiga-group/omiga/src/exchange/shared/repositories"
+	"github.com/omiga-group/omiga/src/shared/enterprise/cron"
+	"go.uber.org/zap"
+)
+
+type KrakenTradingPairsSubscriber interface {
```

Comment:
did you intentionally left this interface as empty,
it looks that `Run` method could go into this interface, right!

or is this used for mock codegen

Reply:
it is used for Wire to inject dependencies, the RUN method is the internal method called by the scheduler. If we ever wanna let others to invoke the RUN for whatever reason outside what it is scheduler for, we can then add it to the interface @sepisoad